addons/mail/static/src/models/composer/composer.js

```diff
@@ -563,7 +563,7 @@ function factory(dependencies) {
             let startIndex = 0;
             for (const match of inputMentions) {
                 const suggestionDelimiter = match[0];
-                const matchName = owl.utils.escape(match.substring(1).replace(new RegExp('\u00a0', 'g'), ' '));
+                const matchName = match.substring(1).replace(new RegExp('\u00a0', 'g'), ' ');
```

Comment:
Please rename the variable to `matchNameEscaped` if this is indeed already escaped from `body`. That will make the code less confusing.